index.html

```diff
@@ -0,0 +1,499 @@
+    <title>laslessVPN</title>
+    <link rel="stylesheet" href="./styles/style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <div class="container">
+        <img
+          src="./img/logo.svg"
+          width="149"
+          height="36"
+          alt="lasslessvpn-logo"
+        />
+        <ul class="header__menu">
+          <li class="header__item">
+            <a class="header__link" href="#">About</a>
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#">Features</a>
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#">Pricing</a>
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#">Testimonials</a>
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#">Help</a>
+          </li>
+        </ul>
+        <div class="container__button">
+          <button class="button button__sign-in" type="button">Sign in</button>
+          <button class="button button__sign-up" type="button">Sign up</button>
+        </div>
+      </div>
+    </header>
+    <main class="main">
+      <section class="main-screen">
+        <h1 class="main-screen__title">
+          Want anything to be easy with
+          <strong class="title--accent">LaslesVPN.</strong>
+        </h1>
+        <p class="main-screen__text">
+          Provide a network for all your needs with ease and fun using
+          <strong class="text--accent">LaslesVPN</strong> discover interesting
+          features from us.
+        </p>
+        <button class="button main-screen__button" type="button">
+          Get Started
+        </button>
+      </section>
+
+      <section class="advantages">
+        <div class="advantages__wrap">
+          <div class="advantages__wrap--item">
+            <img src="./img/user.svg" width="55" height="55" alt="user" />
+            <div class="advantages__text-wrap">
+              <p class="advantages__number">90+</p>
+              <p class="advantages__description">Users</p>
+            </div>
+          </div>
+
+          <div class="advantages__wrap--item">
+            <img
+              src="./img/location.svg"
+              width="55"
+              height="55"
+              alt="location"
+            />
+            <div class="advantages__text-wrap">
+              <p class="advantages__number">30+</p>
+              <p class="advantages__description">Location</p>
+            </div>
+          </div>
+
+          <div class="advantages__wrap--item">
+            <img src="./img/server.svg" width="55" height="55" alt="server" />
+            <div class="advantages__text-wrap">
+              <p class="advantages__number">50+</p>
+              <p class="advantages__description">Servers</p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="features">
+        <img
```

Comment:
Please add a class here — and everywhere else in the doc, an image is also an HTML element and needs to be marked up.

Reply:
+